### DISCUSSION TITLE
Variable global in Method&Body request - (XML - SOAP)

### DISCUSSION THREAD

**Discussion by GiovanniSinosini** [grafana/grafana-infinity-datasource#176] [Q&A] [answered]
How to use global variables in Method&Body requests (XML - SOAP)?
I'm trying to use ${__ from:date :iso} and ${__ to:date :iso}, but it doesn't work.

![image](https://user-images.githubusercontent.com/56783045/138443764-777dc1d0-61ec-4bf6-aaa0-9b0e102d608a.png)

Thank you in advance. This plugin is amazing !!!!

**Comment by yesoreyeram** [comment, marked as answer]
hmm strange.. should work.  ( in explore mode the global variables doesn't work due to a bug in grafana. But in dashboard it should work). 

have tried locally ( replaced URL and BODY with `${__from:date:iso}` and then watch the proxy call body in the developer tools to confirm.

![image](https://user-images.githubusercontent.com/153843/138468090-364209fc-c214-4c31-b6aa-872a0acf57be.png)

Can you confirm you see the same behaviour? Grafana version and plugin version would be useful to debug further.

**Reply by GiovanniSinosini** [reply]
Thank you for your quick reply.   :   )

I will test again.

**Comment by GiovanniSinosini** [comment]
I did more tests again and everything is working.

Thank you very much.

**Reply by yesoreyeram** [reply]
good to know that it is working for you. Wondering what causing the issue before.

**Reply by GiovanniSinosini** [reply]
If I find the reason for the error, I record it here. 

Thank you.   :   )
